ravens/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Based on https://github.com/geyang/jaynes-starter-kit/blob/master/docker/pytorch/Dockerfile
+FROM nvidia/cudagl:11.4.0-devel-ubuntu20.04
+
+FROM python:3.8
```

Comment:
I think this also needs to be deleted. I will patch this up. @takuma-yoneda @Tllokn